wiki/Reporting_bad_behaviour/en.md

```diff
@@ -9,7 +9,7 @@ Accidental reports are ignored. Users who deliberately send invalid reports can
 | Nature of infringement | Point of contact |
 | :-- | :-- |
 | Offensive private messages | **None**: [ignore the user](/wiki/Chat_Console#commands-list) instead. For permanent effect, use the in-game ignore list located in [options](/wiki/Options). |
-| Infringements related to modding and/or mapping | NAT, via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
+| Infringements related to modding and/or mapping | [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) or members of the [GMT](/wiki/People/The_Team/Global_Moderation_Team) focusing on mapping/modding community moderation, via the [NAT/BN Management](https://bn.mappersguild.com/reports) website |
```

Comment:
Does GMT have access to NAT/BN Management website?

Reply:
they do, yeah. i don't think i can share anything here in particular (as they are often confidential) but as a GMT myself i can assure you that GMTs do have access to... *a lot* of mapping-related stuffs on the http://bn.mappersguild.com/ site and elsewhere

Reply:
like for example if you log in into http://bn.mappersguild.com/ as a regular user you'll only be able to see like 3-4 menus at most

if you log in as a member of the GMT however you'll be able to see 11